pkg/azurekeyvault/client/azure_keyvault_service.go

```diff
@@ -98,7 +98,7 @@ func (a *azureKeyVaultService) GetKey(vaultSpec *akvs.AzureKeyVault) (string, er
 }
 
 // GetCertificate download public/private certificates from Azure Key Vault
-func (a *azureKeyVaultService) GetCertificate(vaultSpec *akvs.AzureKeyVault, exportPrivateKey bool) (*Certificate, error) {
+func (a *azureKeyVaultService) GetCertificate(vaultSpec *akvs.AzureKeyVault, exportPrivateKey bool, ensureServerFirst bool) (*Certificate, error) {
```

Comment:
Introduce a option struct, as mentioned earlier?

Reply:
Yes, I agree, will make the changes.